diff -up kio-extras-19.04.0/nfs/CMakeLists.txt.0000~ kio-extras-19.04.0/nfs/CMakeLists.txt
--- kio-extras-19.04.0/nfs/CMakeLists.txt.0000~	2019-04-11 09:05:52.000000000 +0200
+++ kio-extras-19.04.0/nfs/CMakeLists.txt	2019-04-13 15:58:22.849326223 +0200
@@ -1,10 +1,14 @@
 ## Check for XDR functions
 include(CheckFunctionExists)
 
-CHECK_FUNCTION_EXISTS(xdr_u_int64_t HAVE_XDR_U_INT64_T)
-CHECK_FUNCTION_EXISTS(xdr_uint64_t HAVE_XDR_UINT64_T)
-CHECK_FUNCTION_EXISTS(xdr_u_hyper HAVE_XDR_U_HYPER)
-CHECK_FUNCTION_EXISTS(xdr_u_longlong_t HAVE_XDR_U_LONGLONG_T)
+#CHECK_FUNCTION_EXISTS(xdr_u_int64_t HAVE_XDR_U_INT64_T)
+#CHECK_FUNCTION_EXISTS(xdr_uint64_t HAVE_XDR_UINT64_T)
+#CHECK_FUNCTION_EXISTS(xdr_u_hyper HAVE_XDR_U_HYPER)
+#CHECK_FUNCTION_EXISTS(xdr_u_longlong_t HAVE_XDR_U_LONGLONG_T)
+set(HAVE_XDR_U_INT64_T TRUE)
+set(HAVE_XDR_UINT64_T TRUE)
+set(HAVE_XDR_HYPER TRUE)
+set(HAVE_XDR_U_LONGLONG_T TRUE)
 
 if (NOT HAVE_XDR_U_INT64_T AND NOT HAVE_XDR_UINT64_T AND NOT HAVE_XDR_U_HYPER AND NOT HAVE_XDR_U_LONGLONG_T)
     message(FATAL_ERROR "Could not find 64-bit XDR datatype functions!")
@@ -26,7 +30,7 @@ endif (HAVE_XDR_U_LONGLONG_T)
 add_definitions(-DTRANSLATION_DOMAIN=\"kio5_nfs\")
 
 add_library(kio_nfs MODULE kio_nfs.cpp nfsv2.cpp nfsv3.cpp rpc_nfs3_prot_xdr.c rpc_nfs2_prot_xdr.c)
-target_link_libraries(kio_nfs KF5::KIOCore KF5::I18n Qt5::Network)
+target_link_libraries(kio_nfs KF5::KIOCore KF5::I18n Qt5::Network tirpc)
 set_target_properties(kio_nfs PROPERTIES OUTPUT_NAME "nfs")
 
 install(TARGETS kio_nfs DESTINATION ${KDE_INSTALL_PLUGINDIR}/kf5/kio)
