diff -up kio-extras-19.08.3/sftp/CMakeLists.txt.omv~ kio-extras-19.08.3/sftp/CMakeLists.txt
--- kio-extras-19.08.3/sftp/CMakeLists.txt.omv~	2019-11-05 18:39:24.448645840 +0100
+++ kio-extras-19.08.3/sftp/CMakeLists.txt	2019-11-05 18:40:13.823775910 +0100
@@ -20,7 +20,7 @@ target_link_libraries(kio_sftp
    KF5::WidgetsAddons # KMessageBox
    KF5::I18n
    Qt5::Network
-   ${LIBSSH_LIBRARIES})
+   ${LIBSSH_LIBRARIES} ssh)
 set_target_properties(kio_sftp PROPERTIES OUTPUT_NAME "sftp")
 
 install(TARGETS kio_sftp DESTINATION ${KDE_INSTALL_PLUGINDIR}/kf5/kio)
