diff -up kio-extras-21.04.1/smb/autotests/transfertest.cpp.omv~ kio-extras-21.04.1/smb/autotests/transfertest.cpp
--- kio-extras-21.04.1/smb/autotests/transfertest.cpp.omv~	2021-05-12 16:05:46.866805541 +0200
+++ kio-extras-21.04.1/smb/autotests/transfertest.cpp	2021-05-12 16:05:58.811964486 +0200
@@ -6,6 +6,7 @@
 #include <QTest>
 
 #include <future>
+#include <thread>
 
 #include "transfer.h"
 
